src/isBlank.js

```diff
@@ -0,0 +1,44 @@
+import * as R from 'ramda';
+
+import * as RA from '.';
```

Comment:
To avoid circular dependency you have to use following imports:
```suggestion
import isString from './isString';
import isUndefined from './isUndefined';
import isFalse from './isFalse';
```